dev/com.ibm.ws.app.manager.wab/src/com/ibm/ws/app/manager/wab/internal/WABInstaller.java

```diff
@@ -568,9 +577,33 @@ protected boolean uninstallFromWebContainer(WAB wab) {
                             wabGroupsLock.unlock();
                         }
                         if (wab.getCreatedApplicationInfo()) {
+                            // system WABs get uninstalled here
                             deployedApp.uninstallApp(wab);
                         } else {
-                            deployedApp.uninstallModule(wab);
+                            final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
The `latch` is only used inside the following if.  Would be better to only construct it inside the if.